gfx/lz.mk

```diff
@@ -0,0 +1,67 @@
+# Delete these rules if you don't care about matching.
+
+%.lz: LZFLAGS = --method 80
```

Comment:
If you're using the `--align` argument everywhere, use it here, too. Or just remove it.

Reply:
I don't think all rules use `--align`, but we can add an `--align 0` argument to the ones that don't, if that's what you want. Is it?

Reply:
Idk what `--align 0` would do, woudn't `--align 1` just work?

Reply:
Much like RGBASM's own alignment directives, `--align` here specifies the number of low bits that are clear — i.e., it aligns to 1 << n.